selfdrive/controls/controlsd.py

```diff
@@ -88,7 +92,16 @@ def state_control(self):
 
     # Check which actuators can be enabled
     standstill = abs(CS.vEgo) <= max(self.CP.minSteerSpeed, MIN_LATERAL_CONTROL_SPEED) or CS.standstill
-    CC.latActive = self.sm['selfdriveState'].active and not CS.steerFaultTemporary and not CS.steerFaultPermanent and not standstill
+
+    ss_sp = self.sm['selfdriveStateSP']
+    CC.madsEnabled = ss_sp.mads.enabled
+    if ss_sp.mads.available:
+      CC.sunnypilotParams |= SunnypilotParamFlags.ENABLE_MADS.value
+      _lat_active = ss_sp.mads.active
+    else:
+      _lat_active = self.sm['selfdriveState'].active
```

Comment:
Would it be possible to move this away of this file into something like `mads_get_lat_active()` [similar to what we have in panda](https://github.com/sunnypilot/panda/blob/master-new/board/safety.h#L117)? (_Just the abstraction, not the logic of overriding mads lat with controls allowed as we intentionally do it differently here_)